ui/src/core/xibo-forms.js

```diff
@@ -1114,4 +1114,122 @@ function userProfileEditFormOpen() {
             }
         });
     });
+}
+
+function tagsWithValues(formId) {
+    $("#tagValue").addClass("hidden");
+    $('label[for="tagValue"]').addClass("hidden");
+    $('#tagValueRequired').addClass('hidden');
+
+    let tag;
+    let tagWithOption = '';
+    let tagN = '';
+    let tagV = '';
+    let tagOptions = [];
+    let tagIsRequired = 0;
+
+    let formSelector = '#' + formId + ' input#tags';
+
+    $(formSelector).on('beforeItemAdd', function(event) {
+        $('#tagValue').html('');
+        tag = event.item;
+        tagOptions = [];
+        tagIsRequired = 0;
+        tagN = tag.split('|')[0];
+        tagV = tag.split('|')[1];
+
+        if ($(formSelector).val().indexOf(tagN) === -1 && tagV === undefined) {
+            $.ajax({
+                url: $('#'+formId).data().gettag,
+                type: "GET",
+                data: {
+                    name: tagN,
+                },
+                beforeSend: function () {
+                    $("#loadingValues").addClass('fa fa-spinner fa-spin loading-icon')
+                },
+                success: function (response) {
+                    if (response.success) {
```

Comment:
What happens if the response isn't a success - should we have an else here that shows a system error message?